src/pydata_sphinx_theme/__init__.py

```diff
@@ -287,4 +287,4 @@ def setup(app: Sphinx) -> Dict[str, str]:
     # Include component templates
     app.config.templates_path.append(str(theme_path / "components"))
 
-    return {"parallel_read_safe": True, "parallel_write_safe": True}
+    return {"parallel_read_safe": True, "parallel_write_safe": False}
```

Comment:
So for scipy build time is improved even if it's not written in parallel anymore ?

Reply:
yes. on this branch serial scipy build is around 8 minutes on my machine. And with this change it's still possible to use parallel build (active only for the *read* phase) and still shave a few minutes off the build time, without having the sidebar navs come out wrong.